Doppler.ContactPolicies.Data.Access/Entities/ContactPoliciesSettings.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Data.Access.Entities
+{
+    public sealed class ContactPoliciesSettings
+    {
+
+        public User User { get; set; }
+        public bool Enabled { get; set; }
```

Comment:
I think that it will not be used. Right?

Reply:
right now im not using the Enabled field, but in case we need some kind of validation before trying to get the user contact policies if that user has the funtionality ( i know there is a new API with that purpose)

Reply:
Righ, let's talk about it.